### PR TITLE
Update documentation for importing of instances

### PR DESCRIPTION
**Pull request by vishesh92**
To be merged after: https://github.com/apache/cloudstack/pull/9787

----
📚 Documentation preview 📚: https://cloudstack-documentation--453.org.readthedocs.build/en/453/

### CODE REVIEW

**Comment by andrijapanicsb** [comment]
@vishesh92 the other doc PR is for a feature scheduled for 4.20.1 - so not sure this doc PR should wait for that other one ?

**Comment by vishesh92** [comment]
> @vishesh92 the other doc PR is for a feature scheduled for 4.20.1 - so not sure this doc PR should wait for that other one ?

@andrijapanicsb Which PR do you mean? The feature change PR is opened against 4.19 branch as of now.

**Comment by andrijapanicsb** [comment]
If not mistaken, this doc PR should be in here for 4.19.x.x branch, while the other PR (that you said we need to wait for, in the desc of this PR) - https://github.com/apache/cloudstack/pull/9787 - is planned for 4.20.1 (far future).

This PR (453) we need merged already, afaik and should be part of the official 4.19.1.1+ releases.

Or am I wrong @vishesh92

**Review by kiranchavala — COMMENTED** [review]
@vishesh92 

Looks like the formatting is not proper, can you divide it into smaller bullet points or paragraphs.

Also, can you mention the disabled clusters / or hosts which are not enabled or in maintenance mode are not supported for importing vm's

Screenshot 2024-11-13 at 12 35 23 PM

**Comment by kiranchavala** [comment]
@blueorangutan docbuild

File: source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst [hunk 1]
```diff
@@ -113,11 +113,33 @@ In the UI to import instance, you can optionally select a KVM host and temporary
...
+    - Clones the Source Instance on the selected VMware Datacenter for running
+      VMs: The source instance will be cloned in the original state for running
+      VMs. The recommended state is the stopped state to prevent data
+      inconsistencies or loss when cloning the instance.
+    - Imports the VM files (OVF) of the Cloned instance for running VMs, Source
+      Instance for stopped VMs to a temporary storage location (which can be
+      selected by the administrator) from KVM host if ovftool installed or
+      management server (can be forced by the administrator).
+    - Converts the OVF on the temporary storage location to KVM using virt-v2v:
+      CloudStack (or the administrator) selects a running and enabled KVM host to
```
Comment (sureshanaparti) [inline source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst:127]:
```suggestion
      CloudStack (or the administrator) selects a running KVM host to
```

File: source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst [hunk 2]
```diff
@@ -113,11 +113,33 @@ In the UI to import instance, you can optionally select a KVM host and temporary
...
+      store the converted QCOW2 disks of the instance. The disks are then moved
+      into the destination storage pools for the instance. The conversion is a
+      long-lasting process which can be set to time out by the global setting
+      'convert.vmware.instance.to.kvm.timeout'. The conversion processes takes a
+      long time because virt-v2v creates a temporary instance to inspect the
+      source VM and generate the converted disks with the correct
+      drivers. Additionally, it needs to copy the converted disks into
+      the temporary location.
+    - The converted instance is then imported into the selected KVM cluster.
+      The host for conversion is selected randomly from the selected
```
Comment (sureshanaparti) [inline source/adminguide/virtual_machines/importing_vmware_vms_into_kvm.rst:140]:
```suggestion
      The host for import is selected randomly from the selected
```

**Comment by andrijapanicsb** [comment]
@vishesh92 please see my comments in this page.

**Comment by vishesh92** [comment]
> @vishesh92 please see my comments in this page.

Your changes looks good.

**Comment by vishesh92** [comment]
@blueorangutan docbuild

**Comment by vishesh92** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
@blueorangutan docbuild

**Comment by andrijapanicsb** [comment]
@vishesh92 this should be good for merging

**Review by andrijapanicsb — APPROVED** [review]

**Pull request merged**